### ISSUE TITLE
File structure to support multiple workflows

### ISSUE THREAD

**Issue by emiapwil** [openalto/ietf-hackathon#63]
Each workflow has three sets of services

- shared: mininet
- static: rucio/ruciodb/fts/ftsdb/activemq
- dynamic: xrd1-1/..../xrd2-1/...

configuration format:

~~~
name: str # workflow name
static: fts/fts-new -> docker-compose-fts.yml/docker-compose-fts-new.yml -> ...
dynamic:
  hosts:
    ...
  links:
    ...
workload:
  ...
~~~

File structure:

~~~
(CWD): ietf-hackathon

- bin
  |- generate-docker-compose.py [path-to-workflow-config]
  |- analyzer.py [path-to-workflow]
- utils
  |- ...
- etc
  |- ietf-hackathon-114.yml # <- original
- docker
  |- Makefile
  ...
- common
  |- fts
     |- etc
  |- rucio
  |- mininet
     |- create_dynamic_topology.py [path-to-workflow-config]
  \- xrootd
      \- client.conf
- workflow [ignored from git]
  \- test1
     | - docker
         |- static-docker-compose.yml
         \- dynamic-docker-compose.yml
     |- rucio/ruciodb/fts/ftsdb/activemq
        |- etc/rucio or etc/fts  -> /etc/%service%
        |- log/rucio or log/fts  -> /var/log/%service%
     |- sre1
        |- /tmp/*.pem, /tmp/*.key.pem
        |- etc  
           |- xrootd  -> /etc/xrootd
              \- client.conf (copied from common/xrootd
        |- log
           |- xrootd -> /var/log/xrootd
~~~

**Comment by fno2010** [comment]
@jackch996: PR #65 looks good to me as the initial implementation. But it will be great if you can align with the file structure above proposed by @emiapwil

**Comment by emiapwil** [comment]
Solved by PR #68
